go-controller/pkg/ovn/gress_policy.go

```diff
@@ -71,7 +73,7 @@ func (pp *portPolicy) getL4Match() (string, error) {
 	return foundProtocol, nil
 }
 
-func newGressPolicy(policyType knet.PolicyType, idx int, namespace, name string) *gressPolicy {
+func newGressPolicy(policyType knet.PolicyType, idx int, namespace, name string, aclState bool) *gressPolicy {
```

Comment:
I personally dislike boolean arguments on functions - it's quite clear the method does two things.

How about having 2 methods, one for `newGressPolicy` - defaulting to stateful - and another `newStatelessGressPolicy` , which returns a stateless gress policy ?